src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/DataElementStorageTests.cs

```diff
@@ -98,6 +102,44 @@ public void Dispose()
             DeleteApplicationMetadata();
         }
 
+        /// <summary>
+        /// Get data element as org updates download
+        /// </summary>
+        [Fact]
+        public async void Get_DataElement_AsOrg_UpdatesDownloaded_Ok()
+        {
+            if (!blobSetup)
+            {
+                await EnsureValidStorage();
+            }
+
+            DataElement dataElement = (await CreateInstanceWithData(1, false))[0];
+
+            Assert.NotNull(dataElement);
+
+            string dataPathWithDataGuid = $"{versionPrefix}/instances/{testInstanceOwnerId}/{dataElement.instanceGuid}/data/{dataElement.Id}";
+
+            // Get data file once as ORG tests
+            string token = PrincipalUtil.GetOrgToken("tests");
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+            HttpResponseMessage getResponse = await client.GetAsync($"{dataPathWithDataGuid}");
+            getResponse.EnsureSuccessStatusCode();
+            string json = await getResponse.Content.ReadAsStringAsync();
+
+            // Get instance as user to check downloads
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _validToken);
+            getResponse = await client.GetAsync($"{versionPrefix}/instances/{testInstanceOwnerId}/{dataElement.instanceGuid}");
+            getResponse.EnsureSuccessStatusCode();
+
+            Instance instance = JsonConvert.DeserializeObject<Instance>(await getResponse.Content.ReadAsStringAsync());
```

Comment:
Consider using this instead:
`Instance instance = await getResponse.Content.ReadAsAsync<Instance>();`

Reply:
HttpContent does not contain a method for ReadAsAsync.. Is this an extensionmethod defined in a library or in which context have you previously used this notation?